schedule/yast/nfs/yast2_nfs_v3_client.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name:          yast2_nfs_v3_client
+description:    >
+  Multimachine nfs v3 test, client side
+schedule:
+  - {{uefi_bootmenu}}
+  - {{bootloader_zkvm}}
+  - boot/boot_to_desktop
+  - console/yast2_nfs_client
+
+conditional_schedule:
+  uefi_bootmenu:
+    ARCH:
+      aarch64:
+        - boot/uefi_bootmenu
+  bootloader_zkvm:
+    ARCH:
+      s390x:
+        - installation/bootloader_zkvm
```

Comment:
`bootloader_start` should be capable of handling this case, so we can skip having conditional schedule

Reply:
Done